tests/test_operators.py

```diff
@@ -200,6 +200,22 @@ def test_logic(self):
         self.assertEqual(ret['result'], False)
         self.assertEqual(ret['error'], None)
 
+    def test_logic_tensor(self):
+        p = Parser(debug=True)
+        # Numbers
+        ret = p.parse('[3, 2] <> [3, 3]')
+        self.assertEqual(ret['result'], [False, True])
+        self.assertEqual(ret['error'], None)
+        ret = p.parse('4 <> 2')
+        self.assertEqual(ret['result'], True)
+        self.assertEqual(ret['error'], None)
+        ret = p.parse('[1, 2] > [0, 0]')
+        self.assertEqual(ret['result'], [True, True])
+        self.assertEqual(ret['error'], None)
+        ret = p.parse('2 < 3')
+        self.assertEqual(ret['result'], True)
+        self.assertEqual(ret['error'], None)
```

Comment:
Can you maybe move these tests to test_formula_parser.py? I found that the old tests of this repository didn't really work anymore, and only the new test in test_formula_parser.py are being maintained, and are the only ones being run by our just command.

You'll also be able to use `_test_equation` for a more concise form. 

For testing equals, I think you can use an if statement.